src/app/components/users-page/users-page.component.html

```diff
@@ -1,13 +1,18 @@
 <br>
 <h1 class="title"> LISTE DES UTILISATEURS </h1>
-<span class="custom-br"></span>
-<br>
-<span class="d-block p-2 bg-dark text-white user-title"> Matbay Staff </span>
+<hr>
 <br>
 <div class="container">
-	<div class="row g-4">
+	<div class="admin-color blob">
+		<h2 class="user-title">Équipe Technique
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 501.23 20.1"><defs>
+		</defs><g id="Layer_2" data-name="Layer 2">
```

Comment:
Les balises `defs` ne sont pas utilisées.
```suggestion
		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 501.23 20.1"><g id="Layer_2" data-name="Layer 2">
```